login_with/yelp/oauth_10_yelpv2_api.e

```diff
@@ -10,6 +10,8 @@ note
 class
 	OAUTH_10_YELPV2_API
 
+obsolete "On June 30, 2018, API v2 will be discontinued and v2 endpoints will no longer work, move to Yelp Fusion https://www.yelp.com/developers/documentation/v3"
```

Comment:
Not sure this is used by anyone.

I would just clean it up, by removing obsolete service ... or  unpopular one.
Otherwise, we have a lot to maintain.

Reply:
I don't suggest to do the cleaning right now.
But I think we should do that in a near future.

Reply:
Yes, I agree, I have a work in progress branch to do this clean up.